jina/hubble/helper.py

```diff
@@ -76,15 +76,24 @@ def get_request_header() -> Dict:
     :return: request header
     """
     headers = _get_request_header_main()
-    import hubble
 
-    auth_token = hubble.get_token()
+    auth_token = get_token()
     if auth_token:
         headers['Authorization'] = f'token {auth_token}'
 
     return headers
 
 
+def is_logged_in() -> bool: 
+    """Return true if user logged in else false.
+
+    :return: bool  
+    """  
+      
+    auth_token = get_token()
+    return True if auth_token else False
+
+
```

Comment:
It's not accurate to detect if user logged in based on the token existing or not. For example, the token could be there, but it's expired.

I'll create a new function in jina-hubble-sdk and please use that later.

Reply:
Let's use `hubble.is_logged_in()`. `jina-hubble-sdk >= 0.16.1`